source/_integrations/baf.markdown

```diff
@@ -27,6 +27,7 @@ Integrates [Big Ass Fans](https://www.bigassfans.com/) devices into Home Assista
 ## Supported devices
 
 - Haiku Fans with 3.0 firmware and later (Legacy SenseME firmware not supported)
+- Discontinued Haiku Lights with 3.0 firmware and later (Legacy SenseME firmware not supported)
```

Comment:
Here is the light that this is referring to: https://thesmartcave.com/haiku-light-review/

It's a bit confusing since they name everything Haiku

Reply:
Fixed fcc7a5b03a6837f0797f8c8110a35535ba1307ce

Sorry about that